Tests/SentryTests/SentryCrash/SentryCrashReportStore_Tests.m

```diff
@@ -219,4 +220,54 @@ - (void)testStoresLoadsWithUnicodeAppName
     [self expectReports:@[ @(reportID) ] areStrings:@[ reportContents ]];
 }
 
+- (void)test_ScreenshotsPath_forReportId
+{
+    self.appName = @"AppName";
+    [self prepareReportStoreWithPathEnd:@"/ReportPath"];
+    uint64_t reportId = 84568454541;
+
+    char screenshotsPath[SentryCrashCRS_MAX_PATH_LENGTH];
+    sentrycrashcrs_getScreenshotPath_forReportId(reportId, screenshotsPath);
+
+    XCTAssertEqualObjects([NSString stringWithUTF8String:screenshotsPath],
+        [self.tempPath stringByAppendingPathComponent:
+                           @"/ReportPath/AppName-report-00000013b0ac358d-screenshots"]);
+}
+
+- (void)test_ScreenshotsPath_forReport
+{
+    self.appName = @"AppName";
+    [self prepareReportStoreWithPathEnd:@"/ReportPath"];
+
+    uint64_t reportId = 84568454541;
+
+    char reportPath[SentryCrashCRS_MAX_PATH_LENGTH];
+    sentrycrashcrs_getCrashReportPathById(reportId, reportPath);
+
+    char screenshotPath[SentryCrashCRS_MAX_PATH_LENGTH];
+    sentrycrashcrs_getScreenshotsPath_forReport(reportPath, screenshotPath);
+
+    XCTAssertEqualObjects([NSString stringWithUTF8String:screenshotPath],
+        [self.tempPath stringByAppendingPathComponent:
+                           @"/ReportPath/AppName-report-00000013b0ac358d-screenshots"]);
+}
+
+- (void)test_initializeIDs
+{
+    self.appName = @"AppName";
+
+    [self prepareReportStoreWithPathEnd:@"/ReportPath"];
+
+    char firstReportPath[SentryCrashCRS_MAX_PATH_LENGTH];
+    sentrycrashcrs_getNextCrashReportPath(firstReportPath);
+
+    [NSThread sleepForTimeInterval:1];
```

Comment:
`m`: Why do we need the sleep here? Can't we avoid it?

Reply:
😬 This was a test left behind. Should be removed.